cordapp-example/workflows-kotlin/src/main/kotlin/com/example/utils/SumByLong.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.utils
+
+inline fun <T> Iterable<T>.sumByLongSecure(selector: (T) -> Long): Long {
```

Comment:
No longer used.

Reply:
im keeping it as part of learning process - im also learning Kotlin here :)